lib/Account.php

```diff
@@ -42,24 +42,119 @@ class Account extends ApiResource
+    const CODE_VERIFICATION_DOCUMENT_COUNTRY_NOT_SUPPORTED = 'verification_document_country_not_supported';
+    const CODE_VERIFICATION_DOCUMENT_DOB_MISMATCH = 'verification_document_dob_mismatch';
+    const CODE_VERIFICATION_DOCUMENT_DUPLICATE_TYPE = 'verification_document_duplicate_type';
+    const CODE_VERIFICATION_DOCUMENT_EXPIRED = 'verification_document_expired';
+    const CODE_VERIFICATION_DOCUMENT_FAILED_COPY = 'verification_document_failed_copy';
+    const CODE_VERIFICATION_DOCUMENT_FAILED_GREYSCALE = 'verification_document_failed_greyscale';
+    const CODE_VERIFICATION_DOCUMENT_FAILED_OTHER = 'verification_document_failed_other';
+    const CODE_VERIFICATION_DOCUMENT_FAILED_TEST_MODE = 'verification_document_failed_test_mode';
+    const CODE_VERIFICATION_DOCUMENT_FRAUDULENT = 'verification_document_fraudulent';
+    const CODE_VERIFICATION_DOCUMENT_ID_NUMBER_MISMATCH = 'verification_document_id_number_mismatch';
+    const CODE_VERIFICATION_DOCUMENT_ID_NUMBER_MISSING = 'verification_document_id_number_missing';
+    const CODE_VERIFICATION_DOCUMENT_INCOMPLETE = 'verification_document_incomplete';
+    const CODE_VERIFICATION_DOCUMENT_INVALID = 'verification_document_invalid';
+    const CODE_VERIFICATION_DOCUMENT_MANIPULATED = 'verification_document_manipulated';
+    const CODE_VERIFICATION_DOCUMENT_MISSING_BACK = 'verification_document_missing_back';
+    const CODE_VERIFICATION_DOCUMENT_MISSING_FRONT = 'verification_document_missing_front';
+    const CODE_VERIFICATION_DOCUMENT_NAME_MISMATCH = 'verification_document_name_mismatch';
+    const CODE_VERIFICATION_DOCUMENT_NAME_MISSING = 'verification_document_name_missing';
+    const CODE_VERIFICATION_DOCUMENT_NATIONALITY_MISMATCH = 'verification_document_nationality_mismatch';
+    const CODE_VERIFICATION_DOCUMENT_NOT_READABLE = 'verification_document_not_readable';
+    const CODE_VERIFICATION_DOCUMENT_NOT_UPLOADED = 'verification_document_not_uploaded';
+    const CODE_VERIFICATION_DOCUMENT_PHOTO_MISMATCH = 'verification_document_photo_mismatch';
+    const CODE_VERIFICATION_DOCUMENT_TOO_LARGE = 'verification_document_too_large';
+    const CODE_VERIFICATION_DOCUMENT_TYPE_NOT_SUPPORTED = 'verification_document_type_not_supported';
+    const CODE_VERIFICATION_FAILED_ADDRESS_MATCH = 'verification_failed_address_match';
+    const CODE_VERIFICATION_FAILED_BUSINESS_IEC_NUMBER = 'verification_failed_business_iec_number';
+    const CODE_VERIFICATION_FAILED_DOCUMENT_MATCH = 'verification_failed_document_match';
+    const CODE_VERIFICATION_FAILED_ID_NUMBER_MATCH = 'verification_failed_id_number_match';
+    const CODE_VERIFICATION_FAILED_KEYED_IDENTITY = 'verification_failed_keyed_identity';
+    const CODE_VERIFICATION_FAILED_KEYED_MATCH = 'verification_failed_keyed_match';
+    const CODE_VERIFICATION_FAILED_NAME_MATCH = 'verification_failed_name_match';
+    const CODE_VERIFICATION_FAILED_OTHER = 'verification_failed_other';
+
+    const INTERVAL_DAILY = 'daily';
```

Comment:
I don't think this name is right. It should be something like `SETTINGS_PAYOUTS_SCHEDULE_INTERVAL_DAILY` instead? Back to "let's not add any of those" in my mind